tests/cache/tests.py

```diff
@@ -1008,6 +1008,17 @@ def tearDown(self):
         super().tearDown()
         self.drop_table()
 
+    def test_num_queries_of_get_many(self):
+        cache.set_many({'a': 1, 'b': 2})
+        cache.set('expired', 'expired', 0.01)
```

Comment:
I'm not sure if this is working as you expect as I don't think memcache supports fractional seconds, does it?

Reply:
This test runs on database cache only